hortimon/mothership/tickscripts/clone_humidity.json

```diff
@@ -0,0 +1,7 @@
+{
+    "measurement": {"type" : "string", "value" : "relative_humidity" },
+    "where_filter": {"type": "lambda", "value": "\"environment\" == 'clone_incubator'"},
+    "warn": {"type" : "lambda", "value" : "\"mean\" < 50.0 OR \"mean\" > 95.0" },
```

Comment:
these json files define the `warning` and `critical` thresholds for alerts for all our sensors